bootstrap/README.md

```diff
@@ -67,7 +67,48 @@ This section has moved here: https://facebook.github.io/create-react-app/docs/ad
 
 ### Deployment
 
```

Comment:
I suggest to add some quick intro here that would help the reader decide which mode of deployment is suitable for them.

If the user does not have CORS on their domain => the docker way is what will get them up an running fast and easy. 

If they do not want to go the docker way, then the domain must be set with CORS and they can deploy the built app whichever way they want.